source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,326 @@ Sorts Builders
+descending sort orders your data from largest to smallest.
+
+Here are some examples of data sorted in ascending order:
+
+* Numbers: 1, 2, 3, 43, 43, 55, 120
+* Dates: 1990-03-10, 1995-01-01, 2005-10-30, 2005-12-21 
+* Words (Alphabetical): carrot, cheese, cucumber, hummus
+
+Here are some examples of data sorted in descending order:
+
+* Numbers: 100, 30, 12, 12, 9, 3, 1
+* Dates: 2020-01-01, 1998-12-11, 1998-12-10, 1900-07-22 
+* Words (Alphabetical): pear, mango, grapes, apple
+
+The following subsections show how to specify these sorts using
+the ``Sorts`` class.
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()`` static
+factory method. Pass ``Sorts.ascending()``
+the name of the field you need to sort on.
+
+The ``ascending()`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(ascending("<field name>"));
+
+The above ``sort()`` method returns a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection, sorted from smallest
+to largest on the specified field name. 
+
+.. example::
+
+   In this code example, we use the ``ascending()`` method to sort the
+   :ref:`sort_example collection <sorts-builders-sort-example>`  
+   by the ``_id`` field:
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      Bson idSort = ascending("_id");
+      List<Document> results = new ArrayList<>();
+      collection.find().sort(idSort).into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+
+   The output of the code snippet above should look something like this: 
+   
+   .. code-block:: json
+   
+      {"_id": 1, "letter": "c", "food": "coffee with milk"}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+
+Descending
+~~~~~~~~~~
+
+To specify a descending sort, use the ``Sorts.descending()`` static factory
+method. Pass ``Sorts.descending()`` the name of the field you need to sort on.
+
+The following code snippet shows how to specify a descending sort on the
+``_id`` field: 
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("_id"));
+
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   ...
+
+Handling Ties
```

Comment:
Thanks for the note. While I agree this would probably be better in the `fundamentals/crud/read-operations/sort` page, I feel that is expanding the scope of the ticket.

In addition, I feel this does relate to builders, as it explains why `ascending()` and `descending()` take an arbitrary number of string arguments.

If you feel strongly about moving this section as part of this ticket however, I will move it, but I think it will introduce a decent amount of issues and restructuring challenges in the crud/sorts page.

Reply:
I would suggest creating the ticket as I wouldn't expect it to be part of this PR.
I don't see much restructuring necessary as you should be able to reuse your material and combine it with the section of the page starting here:

https://github.com/mongodb/docs-java/blob/master/source/fundamentals/crud/read-operations/sort.txt#L104

The existing Sorts page isn't clearly identifying it as a tie, but it is showing how a secondary sort is applied when there are two matching values using the Builders. Did you have a different view of how you would restructure it and have a list of what issues you'd face?